app/helpers/middleware_domain_helper/textual_summary.rb

```diff
@@ -0,0 +1,27 @@
+module MiddlewareDomainHelper::TextualSummary
+  include TextualMixins::TextualName
+  #
+  # Groups
+  #
+
+  def textual_group_properties
+    %i(name nativeid)
+  end
+
+  def textual_group_relationships
+    # Order of items should be from parent to child
+    %i(ems middleware_server_groups)
+  end
+
+  def textual_group_smart_management
+    %i(tags)
+  end
+
+  def textual_nativeid
+    @record.nativeid
+  end
+
+  def textual_name
+    @record.nativeid
```

Comment:
did you mean `name` and not `nativeid` here? in the beginning of the file there's `include TextualMixins::TextualName` that should already contain an implementation for `textual_name`